projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -84,7 +84,12 @@ export async function showCreateDeploymentDialog() {
       if (runBeforeDeploy) {
         const { message, status } = await jobRunRequest();
         if (status) {
-          await jobRequest('deploy');
+          const success = await jobRequest('deploy');
+          // We only handle the successful deployment scenario.
+          // The failign scenario is handled in the request itself.
+          if(success){
+            alert("Job was run successfully! \n" + success);
```

Comment:
I suggest a more explicit "The test job run completed successfully"

Reply:
done